<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Program</string>
  </property>
  <widget class="QPushButton" name="btnCompute">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radGrayscale">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>106</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radRGB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>106</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
